SoftLayer/CLI/formatting.py

```diff
@@ -333,15 +337,24 @@ class FormattedItem(object):
 
         :param original: raw (machine-readable) value
         :param string formatted: human-readable value
+        :param int length: how long the output should be for this item. Trimmed strings end with ...
     """
 
-    def __init__(self, original, formatted=None):
+    def __init__(self, original='NULL', formatted=None, length=None):
```

Comment:
What do you think about a  "None" value instead of "NULL" ?, In my view, looks better as the default value.

Reply:
Thanks for the input, I'm not really sure why I picked Null honestly.